### DISCUSSION TITLE
AI for Accessibility and Beyond :accessibility: [AI2440M]

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#74970] [Accessibility]
**Speakers:** 
- @edsummersnc (Ed Summers, Head of Accessibility, GitHub)
- Jennifer Mankoff (Richard E. Ladner Professor, Center for Research and Education on Accessible Technology and Experiences (CREATE) / University of Washington)
- Darryl Adams (Director of Accessibility, Intel)
- Joe Devon (Chair, GAAD Foundation)

**Time:** Thursday, Nov 9, 2:45 PM - 3:25 PM PST

> AI has the potential to change the way we learn, work, and play. However, the most profound changes are empowering people with disabilities and expanding human abilities. Join our very own Head of Accessibility, @edsummers and a diverse group of accessibility experts for a whirlwind tour of current and future AI projects that are redefining the limits of human potential.

Thank you for joining the session! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and I'll/we'll do our best to answer.

To watch the recording, please click [`link tbd`](https://github.com/orgs/community/discussions/74970).👈

**Comment by ghostinhershell** [comment]
Anton's (@GreenWizard2015) story of using AI tools like Copilot to gracefully tackle the accessibility barriers he's faced being a software developer who also has cerebral palsy is an inspirational one for sure, full of examples of resilience and ingenuity.
- **Learn more about Anton's story here:** [Coding Accessibility: Anton | Realizing potential with AI](https://www.youtube.com/watch?v=Me2Kc75UiWs) 📺 
  - Also available: [Coding Accessibility: Anton Audio Descriptive Version](https://www.youtube.com/watch?v=XAUvaRdCIa4&t=0s)
- **Check out his full story via The ReadMe Project here:** [Realizing potential with AI](https://github.com/readme/stories/anton-mirhorodchenko) 🚀

**Comment by ghostinhershell** [comment]
From tools like [**BeMyEyes**](https://www.bemyeyes.com/) providing AI powered visual assistance for people who are blind or have low vision, to open source projects like [**Assistive Context-Aware Toolkit (ACAT)**](https://github.com/intel/acat), which enable people with motor neuron diseases and other disabilities to have full access to the capabilities and applications of their computers through very constrained interfaces suitable for their condition. This session has really brought to light all of the ways that the AI revolution can help revolutionize assistive technology for people with disabilities.

!['AI has made me rethink the meaning of disability and its relationship to Artificial Intelligence.' Screenshot from the Universe 2023 presentation.](https://github.com/community/community/assets/14021469/9f558e43-7c62-44cc-9363-9e6c18442d48)

**Comment by nang4ally** [comment]
I'd love the link to the on-demand session once we have those!

**Reply by LayZeeDK** [reply]
https://youtu.be/S40-jPBH820?si=Bf8H2qR5wK_6lvFS

**Reply by ghostinhershell** [reply]
Thank you @LayZeeDK!
